deployment/ansible/roles/nyc-trees.app/templates/nginx-nyc-trees-app.conf.j2

```diff
@@ -6,6 +6,20 @@ server {
 
   access_log /var/log/nginx/nyc-trees-app.access.log logstash_json;
 
+  location /favicon.ico {
```

Comment:
I think that you should be able to delete `src/nyc_trees/apps/core/static/favicon.ico` and replace this entire `location` block with:

```nginx
location = /favicon.ico {
  rewrite /favicon.ico /static/img/favicon/favicon.ico;
}
```